requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.16.2
+pandas==0.24.2
+nltk==3.4
```

Comment:
We had a problem with this, it needs to be`nltk==3.4.5`, did you rebase before pushing?

Reply:
no - my mistake. Should I retract the PR, rebase, then create again?

Reply:
you can rebase then push again same PR

Reply:
ok fixing conflicts now